.github/workflows/setup.yml

```diff
@@ -0,0 +1,29 @@
+name: Development environment setup
+
+# yamllint disable-line rule:truthy
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 4 * * *"
+  push:
+    branches:
+      - dev
+      - rc
+    paths:
+      - "requirements.txt"
+      - "requirements_all.txt"
```

Comment:
We have more requirements and constraint files, this is not enough to cover this.

Reply:
I'm absolutely certain that this is true. Given the utter silence from over a year of reports that your instructions don't work, I created a minimal case PR to prompt you to address this. I couldn't possibly know all of your requirements, because I've never once successfully gotten a working environment by following your documentation, and there's zero published on how your requirements system works, so it's hard to grasp what you want or what you intend.

Reply:
Given that one of your most senior developers also didn't know how your requirements system works, how could anyone else? https://github.com/home-assistant/core/pull/84158#issuecomment-1356631889

It's not a fault of this PR that your author cannot possibly address undocumented requirements. Please take this PR and evolve it to do the intended role of preventing setup from working. Or better yet, please document the system you are upset that I'm not magically aware of.

Reply:
> Please take this PR

I'm not taking your PR. Please adjust it, or, in case you are not planning on moving it forward, please close it and create an issue instead.

> r better yet, please document the system you are upset that I'm not magically aware of.

PRs to improve the documentation are welcome 👍 

Thanks 👍 

../Frenck

Reply:
> PRs to improve the documentation are welcome 👍

 You're forgetting that I tried to improve the documentation, and you were insulting there too, then said that the requirement for accepting documentation improvements was whether Microsoft supports it.